mon-pix/app/utils/progress-in-assessment.js

```diff
@@ -1,6 +1,10 @@
 import ENV from 'mon-pix/config/environment';
 
 function getMaxStepsNumber(assessment) {
+  if (assessment.isPreview) {
+    return 1;
```

Comment:
Pour gagner en lisibilité et casser l'effet magique, j'aurais tendance à nommer ce nombre 

```suggestion
    return PREVIEW_UNIQUE_STEP;
```

A voir s'il y a mieux comme nom